extensions/amp-story-dev-tools/validator-amp-story-dev-tools.protoascii

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
```

Comment:
this is empty, should something be here? it doesn't seem likely.

Reply:
Thanks for the question, this is an early draft that I wanted to use to show the code changes, I'll update all this when it makes more sense. Probably we'll not validate the component because it's lazy loaded by amp-story.